discovery-server/src/test/java/io/airlift/discovery/store/TestEntry.java

```diff
@@ -48,6 +50,6 @@ public void testSerialization()
 
     private static Entry entryOf(String key, String value, long version, long timestamp)
     {
-        return new Entry(key.getBytes(UTF_8), value.getBytes(UTF_8), new Version(version), timestamp, null);
```

Comment:
Add maxAge as a parameter to `entryOf` so it can be tested explicitly. Currently, it's not clear where the MAX_AGE value in `testEntry` comes from.

Reply:
Fixed to add the parameter maxAgeInMs.